src/mdns/mdns_mdnssd.cpp

```diff
@@ -993,14 +994,10 @@ void PublisherMDnsSd::ServiceInstanceResolution::HandleGetAddrInfoResult(DNSServ
     mInstanceInfo.mAddresses.push_back(address);
     mInstanceInfo.mTtl = aTtl;
 
-    otbrLogDebug("DNSServiceGetAddrInfo reply: address=%s, ttl=%u", address.ToString().c_str(), aTtl);
+    otbrLogResultCode(aErrorCode, "DNSServiceGetAddrInfo reply: address=%s, ttl=%" PRIu32, address.ToString().c_str(),
+                      static_cast<unsigned int>(aTtl));
```

Comment:
PRIu32 should match uint32_t rather than unsigned int?

Reply:
Good catch. I converted it by mistake.